soco/core.py

```diff
@@ -1283,7 +1275,15 @@ def add_uri_to_queue(self, uri, position=0, as_next=False):
     @only_on_master
     def add_to_queue(self, queueable_item, position=0, as_next=False):
         """Adds a queueable item to the queue.
-        For Arguments see `add_uri_to_queue`"""
+
+        Args:
+            queueable_item (str): The item to be added to the queue
```

Comment:
Thanks for doing the doc string work as well. Just two detail, the argument type for `queueable_item` is no longer `str` after you moved it. It should now be `DidlObject or MusicServiceItem`.